include/oneapi/dpl/pstl/algorithm_ranges_impl.h

```diff
@@ -1065,7 +1067,7 @@ __pattern_set_difference(__parallel_tag<_IsVector> __tag, _ExecutionPolicy&& __e
     {
         auto __out_last = __parallel_set_op(
             __tag, std::forward<_ExecutionPolicy>(__exec), __first1, __last1, __first2, __last2, __result,
-            [](_DifferenceType1 __n, _DifferenceType2) { return __n; },
```

Comment:
I'm not sure this one is necessary, but it probably makes sense to make it consistent.